samples/dotnet/kernel-syntax-examples/Example12_Planning.cs

```diff
@@ -187,7 +227,7 @@ private static IKernel InitializeKernelAndPlanner(out IDictionary<string, ISKFun
 
         // Load native skill into the kernel skill collection, sharing its functions with prompt templates
         planner = kernel.ImportSkill(new PlannerSkill(kernel), "planning");
-
+        kernel.ImportSkill(new ConditionalSkill(kernel), "conditional");
```

Comment:
I'm wondering if this is how we'll want to do things going forward.

Additionally, I'm still torn on whether we need a special syntax in the prompt for these skills or not -- maybe it could just be like any other. Did you try that?

Reply:
PlannerSkill has a dependency on ConditionalSkill but this is not the ideal way moving forward, the PR proved the concept that the IF behavior worked.

Will think more on changes that don't create this problem of skillname referencing/dependency potentially adding ConditionalSkill as a Planner adittional capability instead of a standalone Skill.